source/engine/taskCreationHelpers.cpp

```
@@ -111,10 +111,10 @@ TF_DEFINE_PRIVATE_TOKENS(
 // Helper function to get the depth compositing setting from the layer settings.
 // \params layerSettings The layer settings to get the depth compositing setting from.
 // \return The depth compositing setting.
-HgiCompareFunction _GetDepthCompositing(BasicLayerParams const* layerSettings [[maybe_unused]])
+HgiCompareFunction _GetDepthCompositing(BasicLayerParams const& layerSettings [[maybe_unused]])
 {
 #ifdef AGP_CONTROLABLE_DEPTH_COMPOSITING
-    return getLayerSettings()->depthCompare;
+    return layerSettings.depthCompare;
```

This is fixing an unrelated inconsistency.
`getLayerSettings()` did not even exist, this code could not compile
(if AGP_CONTROLABLE_DEPTH_COMPOSITING was masking the error)